{"name": "test_login_with_valid_phone_invalid_code", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <Server.API.Test.test_login_api_commercial_trado.TestLogin_API object at 0x00000266A9037220>\n\n    @allure.description('test_login_with_valid_phone_invalid_code')\n    def test_login_with_valid_phone_invalid_code(self):\n        url = LoginConstants.url\n        data = LoginConstants.valid_phone_invalid_code\n        res = requests.post(url, data)\n        res_data = res.json()\n>       assert res.status_code == 400\nE       assert 200 == 400\nE        +  where 200 = <Response [200]>.status_code\n\ntest_login_api_commercial_trado.py:41: AssertionError"}, "description": "test_login_with_valid_phone_invalid_code", "start": 1677093663551, "stop": 1677093664050, "uuid": "b89270b8-1b7b-46da-9bab-ca8a74696c3f", "historyId": "ace1755bc74026fe220cdd6f6042a10d", "testCaseId": "d9a260ed055aa72ef52d808f1c7e36d5", "fullName": "test_login_api_commercial_trado.TestLogin_API#test_login_with_valid_phone_invalid_code", "labels": [{"name": "suite", "value": "test_login_api_commercial_trado"}, {"name": "subSuite", "value": "TestLogin_API"}, {"name": "host", "value": "LAPTOP-CFNBSI63"}, {"name": "thread", "value": "12920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_api_commercial_trado"}]}